### PR TITLE
Add 'Learn more' link to bundle discount summary on Cart and Checkout

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Subj.  Link leads to CMS page with terms and Q&A for bundle discount experiment

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4339?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4339** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4339?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4338** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4338?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4337** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4337?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4336** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4336?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/features/bundleDiscount/BundleDiscountSummary.css.ts [hunk 1]
```diff
@@ -13,3 +13,10 @@ export const bundleDiscountSummary = style({
   padding: theme.space.sm,
   gap: theme.space.sm,
 })
+
+export const learnMoreLink = style({
```
Comment (gustaveen) [inline apps/store/src/features/bundleDiscount/BundleDiscountSummary.css.ts:17]: nit: I see it's a bit inconsistent throughout the site, wondering if it should have an underline as default style

**Review by gustaveen — APPROVED** [review]
👍

**Pull request merged**